lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/driver/ContainerManager.java

```diff
@@ -202,7 +202,7 @@ public void onNext(final Time value) {
     return normalizedRackNames;
   }
 
-  private void init(final Set<String> rackNames) {
+  private void init(final Collection<String> rackNames) {
```

Comment:
yes, but the normalized names are a Collection.
There was a bug. the init was being called with the unnormalized rackNames, instead of the normalized ones. I spotted that in the UT, I fixed it here. Then, it's correct to receive a Collection here.